# backwards configuration, the robot will move backwards
BackwardLocalPlanner:
    k_rho: -1.0
    k_alpha: -0.1
    k_betta: -1.0
    carrot_distance: 0.5 #meters
    carrot_angular_distance: 4 # no constraint, max 3.1416
    xy_goal_tolerance: 0.025
    yaw_goal_tolerance: 0.3
    pure_spinning_straight_line_mode: true
